Week3/homework/app/appRoutes.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+module.exports = app => {
+  const toDoApp = require('./appController');
+
+  // todoList Routes
+  // for a todo
+  app
+    .route('/users/:userId/:toDoListId/:toDoId')
```

Comment:
Non-blocking comment: 

please have a look at the naming of URL in REST interfaces:
https://apiguide.readthedocs.io/en/latest/build_and_publish/use_RESTful_urls.html

Most probably it's something not quite clear at first glance, so I would be happy to explain it to you tomorrow or in Slack if you want.

I'm saying so because you'll need it for your interviews, and its gonna be important for other team members